docs/spec.md

```diff
@@ -1884,7 +1884,7 @@ If `feature_group_count = 1` and `batch_group_count = 1`, then for all
 `output_spatial_index` in the index space of `dim(result, output_spatial_dimensions)`,
 `result[result_shape(:, output_spatial_index, :)] = dot_product` where:
 
-* `padded_lhs = pad(lhs, 0, lhs_padding[:, 0], lhs_padding[:, 1], lhs_base_dilations)`.
+* `padded_lhs = pad(lhs, 0, lhs_padding[:, 0], lhs_padding[:, 1], lhs_base_dilations - 1)`.
```

Comment:
How about `lhs_base_dilations[:] - 1` to explicitly call out all elements are being subtracted by 1.

Reply:
Thanks! It is better.

Reply:
Fwiw my preference lies with `lhs_base_dilations - 1` because it's shorter and looks exactly how this would idiomatically look like in NumPy. I'm fine either way for now, and we'll sort this all out in #289.